### DISCUSSION TITLE
I can't forks from by mobile browser

### DISCUSSION THREAD

**Discussion by baaz421** [community/community#46176] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

From desktop i can forks easily by clicking and makes repo in my account, but when I'm try with in mobile browser i can't able to forks even there is no button for fork
Just fork is appearing when i click it's shows who 
already  fork .
I just want fork function also works in responsive like in mobile view

**Comment by yusancky** [comment]
You can add `/fork` directly to the URL of the repository. 
Using the GitHub repository `python/cpython` (URL `https://github.com/python/cpython`) as an example, forking it should go to `https://github.com/python/cpython/fork`.
